docs/src/RunningOnGPUs.md

```diff
@@ -45,24 +45,25 @@ ITensor currently provides
 package extensions for the following GPU backends:
 
 * [CUDA.jl](https://github.com/JuliaGPU/CUDA.jl) (NVIDIA GPUs)
+* [cuTENSOR.jl] (https://github.com/JuliaGPU/CUDA.jl/tree/master/lib/cutensor) (Submodule of `CUDA.jl`) 
 * [Metal.jl](https://github.com/JuliaGPU/Metal.jl) (Apple GPUs)
 * [AMDGPU.jl](https://github.com/JuliaGPU/AMDGPU.jl) (AMD GPUs)
 
 Our goal is to support all GPU backends which are supported by the [JuliaGPU organization](https://juliagpu.org).
 
+Notice that `cuTENSOR.jl` is an optional contraction backend for `CUDA.jl` and relies primarily on the `CuArray`. If `cuTENSOR` is loaded nearly all `CuArray` based contraction will utilize the `cuTENSOR.mul!` implementation instead of `cuBLAS.mul!`. It is not possible, as of now, to revert to `cuBLAS` if `cuTENSOR` is loaded. Blocksparse support with `cuTENSOR` is an active area of development and has a known bug for certain block-contractions. It is advised as of now to not use `cuTENSOR` with `BlockSparse`.
```

Comment:
I wouldn't describe `cuTENSOR.jl` as a "contraction backend for `CUDA.jl`", rather that it is based on `CUDA.jl` and provides new functionality for accelerated binary tensor contractions.

Also, I think:

> If `cuTENSOR` is loaded nearly all `CuArray` based contraction will utilize the `cuTENSOR.mul!` implementation instead of `cuBLAS.mul!`.

is too low level (and also sounds misleading, it makes it sound like `cuBLAS` has a function `mul!` that performs tensor contraction, which it doesn't). I would just say that when `cuTENSOR` is loaded then ITensors with `CuArray` data are contracted using `cuTENSOR`, while without it contractions are mapped to matrix multiplications which are performed using `cuBLAS`.

Reply:
I updated the docs with your suggestions. Let me know if there is more I should change. Thanks!